### ISSUE TITLE
Wrong focus behavior in the modal window component

### ISSUE THREAD

**Issue by sfi0zy** [sfi0zy/muilessium#11]
If the window opener is focused and the tab key is pressed, keyboard focus falls into an infinite loop. It should be moved to a next focusable element.

**Comment by sfi0zy** [comment]
Fixed in v0.1.33.
